tldr.py

```diff
@@ -30,10 +30,19 @@
     'https://tldr-pages.github.io/assets/tldr.zip'
 )
 
-DEFAULT_LANG = os.environ.get(
-    'LANG',
-    'C'
-).split('_')[0]
+def filter_languages(lang):
+    if lang in ['pt_PT', 'pt_BR', 'zh_TW']:
+        return lang
+    elif lang == "pt":
+        return "pt_PT"
+    else:
+        return lang.split('_')[0]
+
+DEFAULT_LANG = filter_languages(
+    os.environ.get(
+        'LANG',
+        'C'
+    ).split('.')[0])
```

Comment:
Does anyone know how to apply this filter with chain notation for a nicer syntax like
`.split('.')[0].apply(filter_languages)`? 
Couldn't find a solution with a quick search.

Reply:
Sure, that makes sense.